README.md

```diff
@@ -1,56 +1,48 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] [OPTIONAL] Linter should be set up correctly for this repository, kindly add the [linters config files using this repo](https://github.com/microverseinc/linters-config)